babeld/patches/100-local-make-local_kind-function-accessible.patch

```diff
@@ -0,0 +1,39 @@
+From b29cb705c3b717a7d5c61719936464438b9a48f0 Mon Sep 17 00:00:00 2001
+From: Nick Hainke <vincent@systemli.org>
+Date: Fri, 15 Jan 2021 15:01:31 +0100
+Subject: [PATCH] local: make local_kind function accessible
```

Comment:
> Further. we need to add a patch that makes local_kind accessible again: "100-local-make-local_kind-function-accessible.patch"

Why do we need this patch? Did you try to send it to upstream? 

It looks like to me without further looking into it that you are changing something that is not exactly wanted by upstream.

Reply:
https://github.com/jech/babeld/commit/ef3eb65c8615b3dadab7f6f4e2e2383a71d0463f
https://github.com/jech/babeld/commit/fe9407b850dfda4e3c433edfa027bd1192732af8

Reply:
Yes, I see that it is right now included in that repository, but you did not describe it anyhow in the OP. So, yes, it is better to include it instead of blindly adding patches, otherwise, the reviewer needs to spend more time if it is included in the upstream or it isnt. What I only found was this PR https://github.com/jech/babeld/pull/72 2 years ago.